### ISSUE TITLE
Server crashes on launch

### ISSUE THREAD

**Issue by MKwareContributions** [PreyK/Windows-Mobile-Browser-Streaming#2]
Crashes with this error:
```
Unhandled Exception: System.IO.FileLoadException: A procedure imported by 'CefSharp.Core.Runtime.dll' could not be loaded.
   at CefSharp.CefSettingsBase..ctor()
   at CefSharp.OffScreen.CefSettings..ctor() in C:\projects\cefsharp\CefSharp.OffScreen\CefSettings.cs:line 16
   at BrowserServer.Program.Main(String[] margs) in E:\wkspaces\Windows-Mobile-Browser-Streaming\BrowserServer\Program.cs:line 183
```

**Comment by mediaexplorer74** [comment]
> Crashes with this error:
> 
> ```
> Unhandled Exception: System.IO.FileLoadException: A procedure imported by 'CefSharp.Core.Runtime.dll' could not be loaded.
>    at CefSharp.CefSettingsBase..ctor()
>    at CefSharp.OffScreen.CefSettings..ctor() in C:\projects\cefsharp\CefSharp.OffScreen\CefSettings.cs:line 16
>    at BrowserServer.Program.Main(String[] margs) in E:\wkspaces\Windows-Mobile-Browser-Streaming\BrowserServer\Program.cs:line 183
> ```

Maybe, no Chrome browser installed (at server-side) or Chrome version is obsolete... idk... 
Or your "Server" "assembly" (the build that you compiled from sources) uncomplete (check CefSharp.Core.Runtime.dll existence)

**Comment by MKwareContributions** [comment]
I have both Google Chrome and Chromium installed on system.
Also I'm using pre-built server from Releases page.

**Comment by mediaexplorer74** [comment]
> I have both Google Chrome and Chromium installed on system. Also I'm using pre-built server from Releases page.

So strange... Ok, check Google Chrome and Chromium versions.
I noticed that change in last PreyK's commit/push:
![image](https://github.com/PreyK/Windows-Mobile-Browser-Streaming/assets/12032735/37421a23-e4a9-4c15-a23d-d4825845d369)

Cef 112.3.0 -> Cef 113.1.40 . So, new Chromium Embedded framework used  for some dev reasons...  ^ ^

Good luck!

**Comment by MKwareContributions** [comment]
Oh I see.
I'm running Windows 8.1, where Chromium 113 is not supported.
I'll try to run server and Chromium in Windows 10 VM.

**Comment by MKwareContributions** [comment]
Now it works. You need Windows 10 to run the server.

**Comment by mediaexplorer74** [comment]
> Oh I see. I'm running Windows 8.1, where Chromium 113 is not supported. I'll try to run server and Chromium in Windows 10 VM.

8.1? Great OS by Steven Sinofsky. I like that Modern design (Metro codename), indeed. 

> Now it works. You need Windows 10 to run the server.

Cool!! Congradulations.

But I noticed that last Windows-Mobile-Browser-Streaming 2.0 build is very unstable. I forked it and tried to quckly improve stability.. and I have not any success at now. Ok, I'm stopped talking because of it's another story :) 
Nice to virtually meet you, [MKware00](https://github.com/MKware00) )
